react/week2/todo/src/Todo.js

```diff
@@ -0,0 +1,81 @@
+
+import React, { useState } from 'react';
+import Item from './Item';
+import TodoForm from "./TodoForm";
+
+const Todo = () => {
+
+
+	const taskItem = [
+		{
+			text: 'Get out of bed',
+			isCompleted: false
+		},
+
+		{
+			text: 'Brush teeth',
+			isCompleted: false
+		},
+
+		{
+			text: 'Eat breakfast',
+			isCompleted: false
+		},
+	];
+
+	/**
+	 * todos: is taskItem, whose value will be equal to taskItem that we pass in useState() as a parameter
+	 * setTodos is like setState
+	 */
+	const [ todos, setTodos ] = useState( taskItem );
+
+	const addToDo = ( text ) => {
+		const newToDos = [ ...todos, { text } ];
```

Comment:
You leave `isCompleted` undefined here. The code will be slightly more robust if you assume, that `isCompleted` is always a boolean. Basically, you have to add:

```
const newToDos = [ ...todos, { text, isCompleted: false } ];
```